File: 백준/Bronze/5218. 알파벳 거리/알파벳 거리.cc
#include <iostream>
using namespace std;
int main(void) {
    int t, x, y;
    string w1, w2;
    cin >> t;
    for (int i = 0; i < t; i++) {
        cin >> w1 >> w2;
        cout << "Distances: ";
        for (int j = 0; j < w1.size(); j++) {
            x = w1[j] - 65, y = w2[j] - 65;
            cout << (x <= y ? y - x : (y + 26) - x) << ' ';
        }
        cout << "\n";
    }
    return 0;
}

File: 백준/Bronze/5218. 알파벳 거리/알파벳 거리.py
import sys
input=lambda:sys.stdin.readline().rstrip()
for _ in range(int(input())):
    w1,w2=input().split()
    arr=[]
    for i in range(len(w1)):
        x,y=ord(w1[i])-65,ord(w2[i])-65
        if x<=y:
            arr+=[y-x]
        else:
            arr+=[(y+26)-x]
    print("Distances:",*arr)